tests/acceptance/pageObjects/FilesPageElement/publicLinksDialog.js

```diff
@@ -36,12 +36,19 @@ module.exports = {
         .startCase()
         .replace(/\s/g, '')
         .value()
+      const selectedRoleDropdown = util.format(
+        this.elements.publicLinkRoleSelectionDropdown.selector,
+        role
+      )
       return this.waitForElementPresent('@selectRoleButton')
         .click('@selectRoleButton')
         .waitForElementVisible('@rolesDropdown')
         .waitForElementVisible(`@role${role}`)
         .click(`@role${role}`)
         .waitForElementNotVisible('@rolesDropdown')
+        .useXpath()
+        .waitForElementVisible(selectedRoleDropdown)
+        .useCss()
```

Comment:
This is required to check the `role dropdown of public share` changes when a user selects various roles. This asserts that PR https://github.com/owncloud/web/pull/4504   is working as intended.